python_ta/cfg/visitor.py

```diff
@@ -281,6 +303,18 @@ def visit_tryexcept(self, node: nodes.TryExcept) -> None:
                 child.accept(self)
             self._current_cfg.link_or_merge(self._current_block, end_block)
 
+        # Construct the try body so reset current block to this node's block
+        self._current_block = node.cfg_block
+
+        for child in node.body:
+            child.accept(self)
+        end_body = self._current_block
+
+        # Remove each control boundary that we added in this method
+        for _ in range(len(node.handlers) + 1):
```

Comment:
This isn't the correct number if there are multiple exceptions for the same handler

```python
except (RuntimeError, TypeError, NameError):
     ...
```

(note that you have a `for exception in exceptions` loop above)